op-e2e/e2eutils/disputegame/alphabet_helper.go

```diff
@@ -0,0 +1,33 @@
+package disputegame
+
+import (
+	"context"
+
+	"github.com/ethereum-optimism/optimism/op-challenger/config"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/challenger"
+)
+
+type AlphabetGameHelper struct {
+	FaultGameHelper
+	claimedAlphabet string
+}
+
+func (g *AlphabetGameHelper) StartChallenger(ctx context.Context, l1Endpoint string, name string, options ...challenger.Option) *challenger.Helper {
+	opts := []challenger.Option{
+		func(c *config.Config) {
+			c.GameAddress = g.addr
+			c.GameDepth = alphabetGameDepth
+			c.TraceType = config.TraceTypeAlphabet
+			// By default the challenger agrees with the root claim (thus disagrees with the proposed output)
+			// This can be overridden by passing in options
+			c.AlphabetTrace = g.claimedAlphabet
+			c.AgreeWithProposedOutput = false
```

Comment:
There's different semantics between the two - alphabet helper knows the trace that matches the root claim, so it makes sense to inherit that and thus disagree with the proposed output.  Cannon helper doesn't know the trace that matches the root and what will be generated by default is almost certainly going to agree with the proposed output.  But I don't like the inconsistency much either and since we want to be explicit anyway, I'll switch cannon to default to false like the alphabet provider.